features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -323,12 +323,13 @@ const ActionConfigurationPage: FunctionComponent<ActionConfigurationPageInterfac
                     handleError(error, toggleOperation);
                 })
                 .finally(() => {
+                    setIsActive(data.checked);
                     mutateAction();
                     setIsSubmitting(false);
                 });
         };
 
-        return !isLoading && !showCreateForm && (
+        return !isLoading && !showCreateForm && !isEmpty(actions) && (
```

Comment:
do we need !isEmpty(actions) check here as showCreateForm value is determined based on actions object being empty or not

Reply:
This is to tackle the loading the page for the first time, where the isCreatForm has the default false value. And the state change doesnt reflect at that moment.